sdk/workflow.go

```diff
@@ -92,6 +92,23 @@ func (w *Workflow) Nodes() []int64 {
 	return res
 }
 
+//GetNodeByName returns the node given its name
+func (w *Workflow) GetNodeByName(name string) *WorkflowNode {
+	n := w.Root.GetNodeByName(name)
+	if n != nil {
+		return n
+	}
+	for _, j := range w.Joins {
+		for _, t := range j.Triggers {
+			n = t.WorkflowDestNode.GetNodeByName(name)
+			if n != nil {
+				return n
+			}
+		}
+	}
+	return nil
```

Comment:
it could be nice to return a (non blocking ?) error to tell the user that all have not been resynchronized